build.sbt

```diff
@@ -19,13 +19,13 @@ val commonSettings = Seq(
   scalaVersion := "2.13.12",
   crossScalaVersions := Seq("2.12.18", "2.13.12", "3.3.1"),
   libraryDependencies ++= Seq(
-    "com.chuusai" %%  "shapeless" % "2.3.9",
-    "com.typesafe" % "config" % "1.4.2",
+    "com.chuusai" %%  "shapeless" % "2.3.10",
+    "com.typesafe" % "config" % "1.4.3",
     "com.github.pjfanning" %% "pekko-rabbitmq" % "7.0.0",
-    "com.rabbitmq" % "amqp-client" % "5.18.0",
-    "org.slf4j" % "slf4j-api" % "1.7.36",
+    "com.rabbitmq" % "amqp-client" % "5.20.0",
+    "org.slf4j" % "slf4j-api" % "2.0.9",
```

Comment:
slf4j 2 upgrade is a bit of a mess - pekko itself uses slf4j 1 - and there are compatibility issues between the 2 versions

Reply:
Reverted